### PR TITLE
[Enhance] Add CVE inventory subsection to Agents' vulnerability section

### PR DESCRIPTION
**Pull request by AlexRuiz7**
- The new view queries the vulnerabilities of the selected agent and displays the results in a table.
    - The table' data allows sort by columns, enable/disable columns, search and export in CSV format.
    - PDF reporting button has been disabled
    - Table's header for CVE has been capitalized
    - Vulnerability extensions now have a navigation item to move to the new view

### CODE REVIEW

**Comment by AlexRuiz7** [comment]
New commits makes each row of the table expandable (it currently shows the same info as the main row), deletes a duplicated function on the server side table controller and updates the changelog.

This PR depends on [#1112](https://github.com/wazuh/wazuh-splunk/pull/1112), as it fixes a problem with the table where the rows does not expand.

File: SplunkAppForWazuh/appserver/static/js/controllers/agents/cve/agents-cve.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    <!-- Filter bar -->
+    <wazuh-bar></wazuh-bar>
+
+    <div ng-show="loadingVizz">
+      <div align='center'>
+        Fetching data...<br></b><i class="fa fa-fw fa-spin fa-spinner" aria-hidden="true"></i>
+      </div>
+    </div>
+    
+    <div layout="row" class="wz-padding-top-0" ng-if="agent && agent.os && agent.os.platform !== 'windows'">
```
Comment (AlexRuiz7) [inline SplunkAppForWazuh/appserver/static/js/controllers/agents/cve/agents-cve.html]: Shows only on Linux systems. If Windows systems use the exact same view `agent.os.platform != windows` must be removed

**Review by mpRegalado — APPROVED** [review]
CR: LGTM :heavy_check_mark: 
Testing: LGTM :heavy_check_mark: 
Tested the table, pagination, sorting, search and export functions, everything seems in order, great job!

**Review by CPAlejandro — APPROVED** [review]
CR: LGTM!
Testing: LGTM!

**Pull request merged**